upass/__main__.py

```diff
@@ -400,12 +400,17 @@ def call_pass(self, originator, args):
         stdout, stderr = p.communicate()
         if p.returncode == 0:
             self._clear_box()
+            try:
+                copytarget = stdout.decode('utf-8')
+            except AttributeError:
+                copytarget = stdout
             if copy:
-                try:
-                    copytarget = stdout.decode('utf-8')
-                except AttributeError:
-                    copytarget = stdout
-                copytarget = copytarget.split('\n', 1)[0]
+                if type(copy) == type(True):
```

Comment:
This line is kinda ugly; you’d use `if isinstance(copy, bool):` or would avoid it altogether.